Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/DriverPool/Config.hs

```diff
@@ -45,12 +47,19 @@ getDriverPoolConfigFromDB ::
   String ->
   SL.Area ->
   Maybe Meters ->
+  Maybe Seconds ->
   m (Maybe DriverPoolConfig)
-getDriverPoolConfigFromDB merchantOpCityId serviceTier tripCategory area mbDist = do
+getDriverPoolConfigFromDB merchantOpCityId serviceTier tripCategory area mbDist mbDuration = do
```

Comment:
Pool config can also vary with this duration, right?

Reply:
Duration only comes in the picture when we have a trip with greater than 10 hr time(then distance doesn't matter). Otherwise, we can always consider distance